packages/shared/lib/db/config.ts

```diff
@@ -0,0 +1,5 @@
+import { getDbConfig } from './database.js';
+
+const config = getDbConfig({ timeoutMs: 60000 });
+
+export { config };
```

Comment:
Where is this being used?

Reply:
https://github.com/NangoHQ/nango/blob/nan-241-app-oauth-flow/packages/shared/lib/db/knexfile.cjs